doc/tasks/phpstan.md

```diff
@@ -20,6 +20,7 @@ parameters:
             force_patterns: []
             ignore_patterns: []
             triggered_by: ['php']
+            memory_limit: -1
```

Comment:
Probably out of scope of this PR (since the list of options is not sorted already), but what do you think about keeping the list of options sorted by name (also applies to other occurrences in this PR)?

Reply:
Would be nice, but indeed out of scope here.